build/Common.prod.props

```diff
@@ -72,5 +72,8 @@
     <PackageReference Include="StyleCop.Analyzers" Version="1.1.118" Condition="'$(SkipAnalysis)'!='true'">
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="Microsoft.CodeAnalysis.PublicApiAnalyzers" Version="3.0.0" Condition=" $(OS) == 'Windows_NT'">
```

Comment:
will it be enabled on tests as well being placed here?

Reply:
no. tests uses common.test.props